introduction.md

```diff
@@ -1 +1,5 @@
-JavaScript, HTML & CSS
\ No newline at end of file
+JavaScript, HTML & CSS
+
+*Git
+*JavaScript
+*Ruby
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?